packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -63,7 +85,25 @@
       &.is-active {
         background-color: var(--terra-tabs-structural-active-background-color, #fff);
         background-size: var(--terra-tabs-structural-active-background-size, 100% 2px);
+        box-shadow: var(--terra-tabs-structural-active-box-shadow, none);
         color: var(--terra-tabs-structural-active-color, #000);
+        position: relative;
+
+        &::before {
+          transition-duration: var(--terra-tabs-structural-transition-duration, 0);
+          transition-property: width;
```

Comment:
should we open up `transition-property` to be themeable, or constrain it to width?